### DISCUSSION TITLE
sanitize javascript: from string

### DISCUSSION THREAD

**Discussion by 4ch2** [apostrophecms/sanitize-html#548] [Q&A]
Is there a way to use the sanitizer in order remove/escape "javascript:" from a string not inside tags? 
Couldn't find anything in the documentation..

for example:
`dirty = "javascript:alert(1);//"`

**Comment by boutell** [comment]
This is not something that poses a problem as part of the text, as opposed to attributes or script tag body, in HTML. So it is not a supported use case. You might be able to manipulate it with the `transformTags` option, or not (I'm not sure how it'll react if there are no tags at all).
